nextcloud-jail.sh

```diff
@@ -389,28 +423,43 @@ iocage exec "${JAIL_NAME}" su -m www -c 'php /usr/local/www/nextcloud/occ app:en
 iocage exec "${JAIL_NAME}" su -m www -c 'php /usr/local/www/nextcloud/occ encryption:enable'
 iocage exec "${JAIL_NAME}" su -m www -c 'php /usr/local/www/nextcloud/occ encryption:disable'
 iocage exec "${JAIL_NAME}" su -m www -c 'php /usr/local/www/nextcloud/occ background:cron'
+fi
+
 iocage exec "${JAIL_NAME}" su -m www -c 'php -f /usr/local/www/nextcloud/cron.php'
 iocage exec "${JAIL_NAME}" crontab -u www /mnt/includes/www-crontab
 
 # Don't need /mnt/includes any more, so unmount it
 iocage fstab -r "${JAIL_NAME}" "${INCLUDES_PATH}" /mnt/includes nullfs rw 0 0
 
+#####
+#
+# Output results to console
+#
+#####
+
 # Done!
 echo "Installation complete!"
 if [ $NO_CERT -eq 1 ]; then
-  echo "Using your web browser, go to http://${HOST_NAME} to log in"
+  echo "Using your web browser, go to http://${nextcloud_host_name} to log in"
 else
-  echo "Using your web browser, go to https://${HOST_NAME} to log in"
+  echo "Using your web browser, go to https://${nextcloud_host_name} to log in"
```

Comment:
$nextcloud_host_name doesn't seem to be defined--when I run the script, I get this output near the end:
```
Installation complete!
Using your web browser, go to https:// to log in
Default user is admin, password is BX79cBmCvVd4u987
```

Reply:
Shoot, that has been a copy-paste fuckup. Fixed